grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/atom/binary/Relation.java

```diff
@@ -106,7 +106,10 @@ private Relation(Relation a) {
 
     @Override
     public String toString(){
-        return (getType() != null? getType().getLabel() : "") + getRelationPlayers().toString();
+        String relationString = (isUserDefinedName()? "$" + getVarName().getValue() + " ": " ") +
```

Comment:
`VarName#toString` will already prepend the `$`, so this can just be `getVarName() + " "`.